jenkins/bootstrap_test.py

```diff
@@ -1133,6 +1133,14 @@ def testAllJobsHaveErrExit(self):
                      expected, lines,
                      '%s not found in %s' % (expected, job_path))
 
+    def testNoBadVarsInJobs(self):
+        """Searches for jobs that contain ${{VAR}}"""
+        for job, job_path in self.jobs:
+            with open(job_path) as fp:
+                script = fp.read()
+            bad_vars = re.findall("(\${{.+}})", script)
```

Comment:
Please use `'` rather than `"` to stay consistent with the rest of the file.

Also please use `r'foo'` rather than `'foo'` when using a regular expressions to tell python not to escape things like `\$`